galasa-parent/dev.galasa.framework.api.runs/bnd.bnd

```diff
@@ -1,3 +1,5 @@
 -snapshot ${tstamp}
 Bundle-Name: Galasa API Runs microservices
-
+Export-Package: dev.galasa.framework.api.runs;
+Import-Package: dev.galasa.framework.api.common, dev.galasa.framework.api.Authentication, \
```

Comment:
Should this be upper case?

Reply:
let me fix that, just noticed it's also capitalised in the authentication bundle

Reply:
The bnd file for api.authentication is upper case also so they match but unsure if the case would break this :)